### ISSUE TITLE
ManipulateBoneScale doesn't properly scale render boxes for props

### ISSUE THREAD

**Issue by besenyei** [Facepunch/garrysmod-issues#3520]
https://wiki.garrysmod.com/page/Entity/SetModelScale

----------------------------------------------

As soon as you change a props scale, the render box breaks permanently and is always inconsistent with prop sizes.

This works with npcs and players, could you make it work for props too?

I used https://steamcommunity.com/sharedfiles/filedetails/?id=167808759 and r_drawrenderboxes 1 to test.

Even after you revert the model's scale, the render box doesn't revert back to it's normal size.

**Comment by Kefta** [comment]
Does multiplying the render bounds manually fix it?

**Comment by besenyei** [comment]
It would, but It'd be better if the engine did this automatically.

doll with 500% scale.
![20180626190909_1](https://user-images.githubusercontent.com/39219459/41930994-9c563e10-7974-11e8-8edc-41379e3efe33.jpg)

**Comment by robotboy655** [comment]
What exactly is the problem with? props? Why did you link a player resizer addon then?

I also cannot reproduce this.
prop_physics, both singleplayer and local mp tested:
```lua_run print(Entity(1):GetEyeTrace().Entity:SetModelScale(5))```
![image](https://user-images.githubusercontent.com/3299036/42064878-ae579046-7b40-11e8-90d8-dfdc7291cfe2.png)

**Comment by besenyei** [comment]
I apologize, It seems I got it wrong.

I used that addon because it allowed resizing of players, npcs and props with ease.
I tested on a srcds instance of the game, and I can confirm the issue isn't with SetModelScale, so I apologize for making that assumption.

I looked at the addons code, the only things it does to props is SetModelScale, and ManipulateBoneScale.

I guess since this technically counts as a duplicate issue, because it was posted years ago.
https://github.com/Facepunch/garrysmod-issues/issues/1249
